code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -31,8 +31,8 @@
 	..()
 
 /mob/living/simple_animal/cockroach/proc/squish()
-	adjustBruteLoss(1) // kills a normal cockroach
 	new /obj/effect/decal/cleanable/deadcockroach(get_turf(src))
```

Comment:
That's intentional. I considered the case of a human-level intelligence cockroach that has 200 health. It seems unfair to just get suddenly crushed, so if a cockroach gets more than 1 health, it survives fine. See https://youtu.be/L7W2HKtdWRo